CONTRIBUTING.md

```diff
@@ -1,5 +1,90 @@
 Contributing to OpenHMIS
 =================================================================
-Instructions will be added here to explain how to find a list of issues, how to participate in the OpenHMIS community, and how to push new patches.
+The OpenHMIS project is a group effort: thank you for considering helping out!
 
-For now, please view [INSTALL.md](INSTALL.md) for information on how to set up a development environment.
+Step 1: Join the Community
+-------------------
+If you want to contribute, we welcome you to participate in the [OpenHMIS community](http://openhmis.pcni.org/).
+
+**There are three points of access:**
+
+* [The Google Group](http://openhmis.pcni.org/developers/discussion-group) is a forum is for any HMIS topic: administrative, policy/governance, and software development.
+* [Developer Meetings](http://openhmis.pcni.org/developers/developer-meetings) are held on a regular basis, and are generally coordinated via the Google Group.
+* [Github Issues](https://github.com/PCNI/OpenHMIS/issues) are used to keep track of bugs, feature requests, and suggested improvements.
+
+
+Step 2: Review the Issues
+-------------------
+Our [list of Issues](https://github.com/PCNI/OpenHMIS/issues) should have a current list of identified issues with OpenHMIS.  New contributors should start by looking at the [easy pickings](https://github.com/PCNI/OpenHMIS/labels/easy%20pickings) list.
+
+The labels with actionable tasks:
```

Comment:
Discussed with Dan this morning and we'd like to add one additional tag - icebox. This tag will be for long term foundational enhancements that are not immediately actionable but need to not be forgotten. (i.e. cleaning up the repository history)